.reek.yml

```
@@ -114,6 +114,7 @@ detectors:
     enabled: true
     exclude:
     - ExceptionHunter::RequestHunter#call
+    - ExceptionHunter::UserAttributesCollector#collect_attributes
```

Don't exclude it, give the argument a better name

---

if i put anything else rather than `e` it goes for:
```
lib/exception_hunter/collector/user_attributes_collector.rb:11:31: C: Naming/RescuedExceptionsVariableName: Use e instead of error.
      rescue NoMethodError => error
```

and if i put `e` : 
lib/exception_hunter/collector/user_attributes_collector.rb -- 1 warning:
  [11]:UncommunicativeVariableName: ExceptionHunter::UserAttributesCollector#collect_attributes has the variable name 'e' [https://github.com/troessner/reek/blob/v5.6.0/docs/Uncommunicative-Variable-Name.md]

---

https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md
```suggestion
    accept:
      - e
      - ex
```